src/components/main-site/Footer.js

```diff
@@ -85,6 +86,16 @@ const Footer = ({ classes }) => (
                   />
                 </a>
               </span>
+              <span>{""}</span>
+              <span>
+                <a href="https://www.youtube.com/channel/UCh5M0KFUG1-qSYQIUmXHl7w">
+                  {""}
```

Comment:
We can get rid of this line, sometimes prettier does this if it sees spaces inside of tags